app/presenters/return-requirements/check-your-answers.presenter.js

```diff
@@ -4,16 +4,38 @@
  * Formats data for the `/return-requirements/{sessionId}/check-your-answers` page
  * @module CheckYourAnswersPresenter
  */
-
+const { formatLongDate } = require('../base.presenter.js')
 function go (session) {
   const data = {
     id: session.id,
-    licenceRef: session.data.licence.licenceRef
+    journey: session.data.journey,
+    licenceRef: session.data.licence.licenceRef,
+    reason: session.data.reason,
+    startDate: _startDate(session.data)
   }
 
   return data
 }
 
+function _startDate (sessionData) {
+  const selectedOption = sessionData.startDateOptions
+  let date
+
+  if (selectedOption === 'licenceStartDate') {
+    date = new Date(sessionData.licence.currentVersionStartDate)
+  }
+
+  if (selectedOption === 'anotherStartDate') {
```

Comment:
As these are our only two options I think it is okay to just check for one and then assume the other

```suggestion
  } else {
```